src/components/Monster/Monster.jsx

```diff
@@ -6,34 +6,33 @@ import classNames from 'classnames/bind';
 const cx = classNames.bind(styles);
 
 
-export class Monster extends React.Component {
-    render() {
+export function Monster(props) {
```

Comment:
👍 FYI regarding the commit message. It's still a component, it's just a function component instead of a class component.

Reply:
ah okay, thanks